extra_data/tests/test_reader_mockdata.py

```diff
@@ -625,3 +625,13 @@ def test_permission():
         run = RunDirectory(d)
     assert "Permission denied" in str(excinfo.value)
     assert d in str(excinfo.value)
+
+
+def test_empty_file_info(mock_empty_file_valid, capsys):
+    f = H5File(mock_empty_file_valid)
+    f.info()
+    out, err = capsys.readouterr()
+    assert '# of trains:    0' in out
+    assert 'Duration:       0.0' in out
+    assert 'First train ID: -' in out
+    assert 'Last train ID:  -' in out
```

Comment:
I'm not super keen on testing human-readable messages like this, because I don't think we want to guarantee the particular formatting. But so long as there's only a couple such tests, it's not a big deal.

Reply:
what would you suggest instead? Only check the method does not break?

Reply:
That would be my starting point. And then maybe minimal testing of specific bits of the output that are particularly important, or particularly likely to go wrong.

Aside: in case you don't already know it, there's a nice term for tests that only check that something doesn't error: smoketests. Push the button, see if smoke comes out. :slightly_smiling_face:

Reply:
:+1: